es6/Entity.js

```diff
@@ -20,11 +20,23 @@ export class Entity {
 		this._token = token;
 		this._listener = listener;
 	}
-
+	/**
+	 * Checks to see if the entity has className attached.
+	 * @param {*} className Class name you want to see if the entity has attached to it.
+	 */
 	hasClass(className) {
 		return this._entity.hasClass(className);
```

Comment:
this line is buggy, I think you need `this._entity && this._entity.hasClass(className);`, i have it fixed in my newer changes too, if you include it here, i'll remove mine.